editor/project_converter_3_to_4.cpp

```diff
@@ -2803,6 +2807,98 @@ void ProjectConverter3To4::rename_gdscript_functions(String &file_content) {
 			}
 		}
 
+		// -- r.intersect_ray( a, b, c, d, e )  ->  var ray_query = ... ray_query.from = from ... r.intersect_ray(ray_query)         PhysicsDirectSpaceState2D & PhysicsDirectSpaceState3D
```

Comment:
This will change after #61918 is merged.

Reply:
Great, didn't know there's a WIP for this. It surely will be much more convenient to have it as a one-liner rather than the current method of creating a separate object. Should we disregard this conversion and wait for #61918?